package.json

```diff
@@ -2,6 +2,13 @@
   "name": "tutanota",
   "version": "3.25.11",
   "scripts": {
+    "pretest": "npm run-script flow",
+    "test": "npm run-script testapi && npm run-script testclient",
```

Comment:
I see something similar to this included in [merged code](https://github.com/tutao/tutanota/commit/0708dae2#diff-b9cfc7f2cdf78a7f4b91a753d10865a2R12).